.github/workflows/template-update-notification.yml

```
@@ -0,0 +1,86 @@
+name: Check diff between latest commits and create an issue if the is a difference.
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - templates/**
+
+  workflow_dispatch:
+
+jobs:
+
+  check-diff:
+    runs-on: ubuntu-latest
+
+    outputs:
+      diff: ${{ steps.get-diff.outputs.diff }}
+      file: ${{ steps.find-changes.outputs.file_changed }}
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+
+    - name: Find file changed
+      id: find-changes
+      run: |
+        git fetch origin main
+        echo file_changed=$(git diff --name-only HEAD^ HEAD | grep '^templates/') >> $GITHUB_OUTPUT
```

```suggestion
        echo file_changed=$(git diff --name-only HEAD^ HEAD | grep '^templates/' | sed -r `s|templates/||`) >> $GITHUB_OUTPUT
```

https://unix.stackexchange.com/questions/492197/how-to-remove-first-segment-or-segments-from-path-in-shell-script